client/tests/integration/smartcontracts/multisig_register/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "multisig_register"
+
+edition.workspace = true
+version.workspace = true
+authors.workspace = true
+
+license.workspace = true
+
+[lib]
+crate-type = ['cdylib']
+
+[dependencies]
+iroha_trigger.workspace = true
+iroha_executor_data_model.workspace = true
+
+panic-halt.workspace = true
+lol_alloc.workspace = true
+getrandom.workspace = true
+
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true, default-features = false }
+
+[build-dependencies]
+iroha_wasm_builder = { version = "=2.0.0-pre-rc.21", path = "../../../../../wasm_builder" }
```

```suggestion
iroha_wasm_builder = { workspace = true }
```

should we bring it into workspace?

---

not sure, it's used only here now